app/assets/stylesheets/application.css

```diff
@@ -33,3 +33,75 @@
   align-items: center;
   padding-left: 2rem;
 }
+
+.public-container {
+  display: grid;
+  grid-template-columns: 20% 20%;
+  row-gap: 3rem;
+  border: 2px solid black;
+  width: 50%;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 1rem;
+}
+
+.header-title {
+  text-align: center;
+}
+
+.food-list {
+  text-align: center;
+}
+
+.table {
+  /* border: 1px solid black; */
```

Comment:
@Optional...
It would be great practice to remove unnecessary commented lines to make your code cleaner.

Reply:
yes sure